content/docs/reference/reproducibility.md

```diff
@@ -0,0 +1,41 @@
++++
+title="Reproducible Builds"
+weight=9
++++
+
+## Summary
+The Cloud Native Buildpacks project aims to create "Reproducible Builds" of container images. For image creation commands (`create-builder`, `package-buildpackag`, `build`) `pack` aims to create in a reproducible fashion. "Reproducible" is hard to define but we'll do so by example:
+
+---
+Running `pack build` produce a container image with the same image ID (*local* case)
+
+**Given**:
+- A workspace directory containing the same source code
+- A builder image with a *given*  
+- One or more buildpacks that produce identical layers given their input*
+
+---
+Running `pack build --publish` produce a container image with the same image digest (*remote* case)
```

Comment:
produce > produces

Reply:
You say "image digest" here and "image ID" above - are these the same? It also might be worth clarifying how a user can find these values (e.g., `docker inspect`).

Reply:
I see now that these are indeed intended to be different... still I think some additional info on how to view / verify these values would be helpful.